charts/argo-cd/README.md.gotmpl

```diff
@@ -89,10 +92,10 @@ Helm cannot upgrade custom resource definitions [by design](https://helm.sh/docs
 Please use `kubectl` to upgrade CRDs manually from [crds](crds/) folder or via the manifests from the upstream project repo:
 
 ```bash
-kubectl apply -k https://github.com/argoproj/argo-cd/manifests/crds\?ref\=<appVersion>
+kubectl apply -k "https://github.com/argoproj/argo-cd/manifests/crds?ref=<appVersion>"
 
-# Eg. version v2.4.2
-kubectl apply -k https://github.com/argoproj/argo-cd/manifests/crds\?ref\=v2.4.2
+# Eg. version v2.4.9
+kubectl apply -k "https://github.com/argoproj/argo-cd/manifests/crds?ref=v2.4.9"
```

Comment:
Looks good and not a blocker, but as @mkilchhofer already commented - bumping the version here doesn't make much sense, i.e. current `main` is on 2.4.10 already and we want to avoid always bumping this with Chart bumps.

Reply:
I've reverted the change but proposal was to bump this together with appVersion via template. We can add it in future if we want.